zfs-prune-snapshots

```diff
@@ -1,12 +1,12 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
`bash` in not in `/bin` on all systems with ZFS

Reply:
Ah, sorry, I just reverted it, I use teh script under FreeBSD

Reply:
I did not aim to deeply modify your script, but using `#!/usr/bin/env` can be useful under perl or python to specify specific versions for example, but for bash, I am not sure it is a good way + the potential security risks depending on the plateforms / use. But it is another debate like using `getopts` instead of a manual parser with input sanitization. Anyway, hope you merge the fixes because the recursiveness wasn't really ideal: no way to not prune recursively, and the unexpected deletes of cloned snapshots using `-R`